### PR TITLE
Remove metadata.description from show-readme.yaml

### PR DESCRIPTION
**Pull request by pavanstarmanwar**
# Changes

show-readme.yaml file contains description which causes following error since description under metadata is an unknown field, removing that line fixed the issue.

Failure:
```
kubectl apply -f show-readme.yaml
Error from server (BadRequest): error when creating "show-readme.yaml": Task in version "v1beta1" cannot be handled as a Task: strict decoding error: unknown field "metadata.description"
```
Success:

```
kubectl apply -f show-readme.yaml
task.tekton.dev/show-readme created
```

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/website/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Review by AlanGreene — CHANGES_REQUESTED** [review]
Thanks for the PR @pavanstarmanwar. I think we should keep the description, just move it to the correct location: `spec.description`

**Comment by pavanstarmanwar** [comment]
> spec.description

Done @AlanGreene

**Comment by pavanstarmanwar** [comment]
@AlanGreene updated , thanks again

File: content/en/docs/How-to-guides/clone-repository.md [hunk 1]
```diff
@@ -493,10 +493,10 @@ apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
   name: show-readme
-  description: Read and display README file.
 spec:
+  description: Read and display README file.
   workspaces:
-  - name: source
+  - name: source
```
Comment (AlanGreene) [inline content/en/docs/How-to-guides/clone-repository.md:499]: nit: please remove the trailing whitespace

**Review by AlanGreene — APPROVED** [review]
Thanks @pavanstarmanwar, just 1 minor thing and it's good to go 👍

**Pull request merged**